src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -26,6 +26,7 @@
 [assembly: InternalsVisibleTo("OrleansProviders")]
 [assembly: InternalsVisibleTo("OrleansHost")]
 [assembly: InternalsVisibleTo("OrleansAzureUtils")]
+[assembly: InternalsVisibleTo("OrleansServiceBus")]
```

Comment:
We went through great efforts to ensure that the EHSP used public capabilities and was a working example of how external users could develop stream providers for other queues.  Basically if we can't develop a persistent recoverable without the need of internals than their is something wrong with our public surface.